code/modules/vending/wardrobes.dm

```diff
@@ -278,7 +278,6 @@
 					/obj/item/clothing/under/rank/chef = 3,
 					/obj/item/clothing/under/rank/altchef = 3,
 					/obj/item/clothing/under/plasmaman/chef = 1,
-					/obj/item/clothing/head/helmet/space/plasmaman/white = 1,

```

Comment:
What's this being removed for?

Reply:
to be honest I don't know why but it keeps showing up in my PRs even though I don't put it, it goes across branched and this line was already removed. 

But either way its not ment to be in the vendors since I made a PR to add replacement helms, if you scroll down like 2 more lines you'll see the replacement helmet